src/cozmo/faces.py

```diff
@@ -341,18 +348,20 @@ def _recv_msg_robot_erased_enrolled_face(self, evt, *, msg):
 
     #### Commands ####
 
-    def name_face(self, name):
+    def name_face(self, name, play_anim=True):
         '''Assign a name to this face. Cozmo will remember this name between SDK runs.
 
         Args:
             name (string): The name that will be assigned to this face
-
+            play_anim (bool): True plays a scanning animation,
```

Comment:
what's a "scanning animation"?  whatever it is, i think it needs a clearer description.. if there's not space here, then perhaps add it to the module and/or class docs